deno.json

```diff
@@ -0,0 +1,6 @@
+{
+  "tasks": {
+    "check": "deno run --allow-read --allow-net https://deno.land/x/link_checker/check.ts",
```

Comment:
Sorry, I'm ignorant of `deno.json`, so this is mostly a note for me to look up myself, but: should this use a relative path to check.ts instead of pointing to "the latest version on deno.land/x"?

Reply:
The docs appear to use a relative path: https://deno.land/manual@v1.31.2/getting_started/configuration_file

Using a relative path seems like the right thing to do here, but as noted earlier, this is the first time I've used a deno.json file.

What do you think, @hashrock ?

Reply:
This should most definitely be a relative path, nice catch.